extensions/amp-story-auto-ads/0.1/story-ad-ui.js

```diff
@@ -223,22 +290,10 @@ export function createCta(doc, buttonFitter, container, uiMetadata) {
       return null;
     }
 
-    const ctaLayer = doc.createElement('amp-story-cta-layer');
-    ctaLayer.className = 'i-amphtml-cta-container';
-
-    const linkRoot = createElementWithAttributes(
-      doc,
-      'div',
-      dict({
-        'class': 'i-amphtml-story-ad-link-root',
-        'role': 'button',
-      })
-    );
-
-    createShadowRootWithStyle(linkRoot, a, ctaButtonCSS);
-
-    ctaLayer.appendChild(linkRoot);
-    container.appendChild(ctaLayer);
-    return a;
+    if (isExperimentOn(doc.defaultView, 'amp-story-page-outlink')) {
```

Comment:
nit: lets name the experiment something so that it's obvious its ads

Reply:
I think you might have missed this one

Reply:
I didn't, it seemed to create some problems with some tests if I changed it for some reason? So it would take a while to refactor. As it's a nit, I just left it as is.

Reply:
Important nit :) please change. You might have needed to change the  name in `toggleExperiment()` for your tests as well.

Reply:
okay changed =)